### PR TITLE
Verify Spectrum Scale Version for Snapshot support

### PR DESCRIPTION
**Pull request by madhuthorat**
Signed-off-by: Madhu Thorat <madhu.punjabi@in.ibm.com>

### CODE REVIEW

File: driver/csiplugin/connectors/rest_v2.go [hunk 1]
```diff
@@ -179,6 +179,25 @@ func (s *spectrumRestV2) GetTimeZoneOffset() (string, error) {
...
 
+func (s *spectrumRestV2) GetScaleVersion() (string, error) {
+	glog.V(4).Infof("rest_v2 GetScaleVersion")
+
+	getVersionURL := utils.FormatURL(s.endpoint, "scalemgmt/v2/info")
+	getVersionResponse := GetInfoResponse_v2{}
+
+	err := s.doHTTP(getVersionURL, "GET", &getVersionResponse, nil)
+	if err != nil {
+		glog.Errorf("Unable to get Spectrum Scale version: [%v]", err)
```
Comment (deeghuge) [inline driver/csiplugin/connectors/rest_v2.go:190]: error strings should not be capitalized

File: driver/csiplugin/connectors/rest_v2.go [hunk 2]
```diff
@@ -179,6 +179,25 @@ func (s *spectrumRestV2) GetTimeZoneOffset() (string, error) {
...
+		return "", err
+	}
+
+	if len(getVersionResponse.Info.ServerVersion) == 0 {
+		return "", fmt.Errorf("Unable to get Spectrum Scale version.")
```
Comment (deeghuge) [inline driver/csiplugin/connectors/rest_v2.go:195]: error strings should not be capitalized

File: driver/csiplugin/connectors/rest_v2.go (hunk 1 above)
Comment (madhuthorat) [reply driver/csiplugin/connectors/rest_v2.go]: > error strings should not be capitalized

Done

File: driver/csiplugin/connectors/rest_v2.go (hunk 2 above)
Comment (madhuthorat) [reply driver/csiplugin/connectors/rest_v2.go]: Done

**Review by deeghuge — APPROVED** [review]
lgtm

**Pull request merged**